draft-ietf-rtcweb-jsep.xml

```diff
@@ -2196,11 +2197,14 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                         specified in <xref target="RFC5245"></xref>,
                         Section 15.4.</t>
 
-                        <t>An "a=fingerprint" line for each of the
-                        endpoint's certificates, as specified in <xref
-                        target="RFC4572"></xref>, Section 5; the digest
-                        algorithm used for the fingerprint MUST match
-                        that used in the certificate signature.</t>
+                        <t>One or more "a=fingerprint" line(s) for
+                        each of the endpoint's certificates, as
+                        specified in <xref
+                        target="I-D.ietf-mmusic-4572-update"></xref>.
+                        At least one of the "a=fingerprint" lines for
+                        each certificate MUST have use the digest
+                        algorithm used in the certificate
+                        signature.</t>
```

Comment:
It's not in 4572 update yet, but I think this is all going to end up there. What do you think of just remove the part with "At least one of the "a=fingerprint" lines for
 +                        each certificate MUST have use the digest
 +                        algorithm used in the certificate
 +                        signature" and leaging that to 4572-update.